hs_auth_minio/api/routers/minio.py

```diff
@@ -165,9 +165,8 @@ def _check_user_authorization(user_id, resource_id, action, is_md_path):
     enable_edit_actions = os.environ.get(
         "ENABLE_EDIT_ACTIONS", "false").lower() == "true"
     if enable_edit_actions and action in EDIT_ACTIONS:
-        if is_md_path:
-            # if the prefix request is a metadata path, we do not allow edit
-            # access
+        if not is_contents_path:
```

Comment:
The md path determined here isn't relevant anymore and it makes more sense to only allow users write access to data/contents/ path so I modified it here.